### ISSUE TITLE
Backlash

### ISSUE THREAD

**Issue by Mgilbride** [winder/Universal-G-Code-Sender#132]
Any thoughts of including backlash compensation feature within gui?

**Comment by optimizedsms** [comment]
I see it is a planned-enhancement.   Any suggestions for those of us who are anxious to give it a try?
I notice preprocessCommand has changed a bit.  Is that still the right place if I want to take a stab?
Thanks.

**Comment by winder** [comment]
@optimizedsms Yes, preprocessCommand is still a good place to start. Take a look at how the `convertArcsToLines` feature works, I think backlash compensation would be similar.

**Comment by khturnings** [comment]
I see that someone has had a go at adding backlash compensation at the controller level - has anyone with a programming skill looked at it - https://github.com/gnea/grbl/issues/661
It seems that it is currently only available for an Arduino Mega, anyone fancy a go at getting a UNO version working or is this not possible?

**Comment by breiler** [comment]
I have added a backlash compensation in #3178.
I would probably not run this on an expensive machine... It was ok on my toy plotter.
